README.md

```diff
@@ -99,6 +100,10 @@ docker run \
 
 Precompiled binaries for the latest release can be found [here](https://github.com/launchdarkly/ld-find-code-refs/releases/latest). Be sure to install the required [dependencies](#prerequisities) before running `ld-find-code-refs`.
 
+### Using code references in federal environments
+
+If you are using the FedRAMP compliant [LaunchDarkly federal instance](https://docs.launchdarkly.com/home/advanced/federal), the `ld-find-code-refs` binary should be compiled with FIPS 140-2 support by using a tool like [boringssl](https://github.com/golang/go/tree/dev.boringcrypto/misc/boring#caveat).
```

Comment:
I don't think we need to link to the caveat anchor here; the full page should be relevant. Also `Go+BoringCrypto` seems to be what they call the project (boringssl is a slightly different project; this is the Golang integration of BoringSSL-- this is probably a distinction without a difference, but since this is public, we should try to call it by the official name).

```suggestion
If you are using the FedRAMP compliant [LaunchDarkly federal instance](https://docs.launchdarkly.com/home/advanced/federal), the `ld-find-code-refs` binary should be compiled with FIPS 140-2 support by using a tool like [Go+BoringCrypto](https://github.com/golang/go/tree/dev.boringcrypto/misc/boring).
```

Reply:
thanks for the feedback!